### PR TITLE
Fix eventsub_websocket example 

### PR DESCRIPTION
**Pull request by gamahead**
The `max_write_buffer_size: 2048` looks like it might have been a mistake but it's definitely wrong. I'm not sure what the original intention was, but I don't think it matters for a simple example so I'm letting it take the default. Lemme know if you want something else there. 

For the undocumented new field on the welcome message, idk

### CODE REVIEW

File: src/eventsub/event/websocket.rs [hunk 1]
```diff
@@ -62,6 +62,9 @@ pub struct SessionData<'a> {
     /// The UTC date and time that the connection was created.
     #[serde(borrow = "'a")]
     pub connected_at: Cow<'a, types::TimestampRef>,
+    /// Undocumented field
+    #[serde(borrow = "'a")]
+    pub recovery_url: Option<Cow<'a, str>>,
```
Comment (Nerixyz) [inline src/eventsub/event/websocket.rs:67]: Could you mention what this is for? Especially in comparison to `reconnect_url`. There's also the `unsupported` feature - not sure if this should be used here.

Comment (gamahead) [reply src/eventsub/event/websocket.rs:67]: I'd love to but I have no idea what it's for. I'll do some digging

Comment (gamahead) [reply src/eventsub/event/websocket.rs:67]: @Nerixyz twitch dev on discord said he doesn't know what it does and can be ignored. Idk what proper resolution is here so let me know what you think. What is the `unsupported` feature you speak of?

Comment (Emilgardis) [reply src/eventsub/event/websocket.rs:67]: Barry is not a twitch developer, he develops stuff using twitch ;)

Maybe lets just put this field `#[doc(hidden)]`.

> There's also the unsupported feature - not sure if this should be used here.

No that would defeat the purpose of the pr to fix deny unknown fields.


```suggestion
    #[doc(hidden)]
    #[serde(borrow = "'a")]
    pub recovery_url: Option<Cow<'a, str>>,
```

**Review by Emilgardis — APPROVED** [review]

**Comment by Emilgardis** [comment]
thank you!

**Pull request merged**